package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@dellstorage/dell-design-react-common",
   "description": "Override CSS of Clarity-React components to align it with Dell design standards",
-  "version": "0.1.4",
+  "version": "0.1.5",
```

Comment:
Please update version of clarity react lib as well

Reply:
Version of clarity react was already set to 1.1.13 for this repo hence didn't update it and yarn.lock is updated now to take 1.1.14

Reply:
ok